tsd/search_scihub.py

```diff
@@ -93,6 +93,7 @@ def build_scihub_query(aoi, start_date=None, end_date=None,
         query += ' AND sensoroperationalmode:{}'.format(operational_mode)
     query += ' AND beginposition:[{}Z TO {}Z]'.format(start_date.isoformat(),
                                                       end_date.isoformat())
+    query += 'AND relativeorbitnumber:{}'.format(relative_orbit_number)
```

Comment:
I think this line will crash if the parameter `relative_orbit_number` is `None`. We should add it to the query only if it's not `None`. Could you add an `if` to fix that?